### PR TITLE
Fixes #725-Fix footer and ensure complete responsiveness

### LINKED ISSUE #725: Fix footer and ensure complete responsiveness
In small resolutions the footer disappears completely. The logo and menu items are not aligned in all resolutions. See more issues below in screenshots.

screenshot from 2017-08-11 19-30-03

screenshot from 2017-08-11 19-33-41

screenshot from 2017-08-11 19-33-18

screenshot from 2017-08-11 19-35-19

### PR DESCRIPTION
**Pull request by rishiraj824**
Fixes issue #725

Changes: 
- Fixed footer  responsiveness
- Refactored and removed the redundant CSS

Demo Link:
http://susi-footer.surge.sh

Screenshots for the change: 
localhost-42366-overview nexus 4
screenshot from 2017-08-14 02 48 55

### CODE REVIEW

File: .com.google.Chrome.9Ei5Hn/SingletonCookie [hunk 1]
```diff
@@ -0,0 +1 @@
+13571232215521641445
```
Comment (isuruAb) [inline .com.google.Chrome.9Ei5Hn/SingletonCookie]: could you please explain this line ?

Comment (rishiraj824) [inline .com.google.Chrome.9Ei5Hn/SingletonCookie]: will remove these files its a typo

**Comment by rishiraj824** [comment]
@mariobehling Please check, added left spacing to the bottom logo, updated the latest code and the link :+1:  http://susi-footer.surge.sh

**Comment by uday96** [comment]
The footer in desktop looks like : 

foot

Is this expected? Shouldn't the the links be aligned to the left or right completely?

**Comment by rishiraj824** [comment]
susi-footer surge sh-overview laptop with hidpi screen
@uday96 Check now on your screen

**Comment by mariobehling** [comment]
Thank you.

**Pull request merged**